src/Mod/Part/App/FT2FC.cpp

```diff
@@ -155,15 +155,15 @@ PyObject* FT2FC(const Py_UNICODE *PyUString,
 //  This sets size to 48 point. Magic.
     error = FT_Set_Char_Size(FTFont,
                              0,             /* char_width in 1/64th of points */
-                             48*64,         /* char_height in 1/64th of points */
+                             480*64,         /* char_height in 1/64th of points */
                              0,             /* horizontal device resolution */
                              0 );           /* vertical device resolution */
     if (error) {
         ErrorMsg << "FT_Set_Char_Size failed: " << error;
         throw std::runtime_error(ErrorMsg.str());
     }
 
-    scalefactor = stringheight/float(FTFont->height);
+    scalefactor = (stringheight/float(FTFont->height))/10;
```

Comment:
Probably worth adding a comment here explaining `/10`

Reply:
added comments to explain the changes, and added the *10 term for clarity (instead of just increasing the existing value)